gum/backend-darwin/gumprocess-darwin.c

```diff
@@ -169,11 +169,85 @@ struct _DyldImageInfo64
   guint64 image_file_mod_date;
 };
 
-#ifndef PROC_SETPC_NONE
-extern int proc_regionfilename (int pid, uint64_t address, void * buffer,
-    uint32_t buffersize);
+#ifndef PROC_INFO_CALL_PIDINFO
+
+#define PROC_INFO_CALL_PIDINFO 0x2
+#define PROC_PIDREGIONPATHINFO 8
```

Comment:
Nested `#define`s should be indented by one space for each level. So here that means:

```C
# define PROC_INFO_CALL_PIDINFO 0x2
# define PROC_PIDREGIONPATHINFO 8
```